tests/HeaderTest.php

```diff
@@ -68,4 +68,13 @@ public function testParsesArrayHeaders(): void
         $header = ['a, b', 'c', 'd, e'];
         self::assertSame(['a', 'b', 'c', 'd', 'e'], Psr7\Header::normalize($header));
     }
+
+    public function testNormalizeWithPcreBackTrackLimited(): void
+    {
+        $limited = ini_get('pcre.backtrack_limit');
+        ini_set('pcre.backtrack_limit', '4');
+        $things = ['"Google Chrome";v="95", "Chromium";v="95", ";Not A Brand";v="99"'];
+        self::assertSame(['"Google Chrome";v="95", "Chromium";v="95", ";Not A Brand";v="99"'], Psr7\Header::normalize($things));
```

Comment:
This assertion is bogus. However the backtrack limit is a good reason to not rely on regular expressions here. See: #447

Reply:
Agree. Rely on the regular expression may not a good idea. For example, parsing the new [Permissions-Policy](https://github.com/w3c/webappsec-permissions-policy/blob/main/permissions-policy-explainer.md#new-header)'s value, eg: `Permissions-Policy: fullscreen=(self "https://example.com" "https://another.example.com"), geolocation=*, camera=()`, there were the parentheses, `self`, `none`, empty string and double quotes strings, it shall be so hard to do that.